docs/source/references/schema.json

```diff
@@ -1043,6 +1050,107 @@
       ]
     }
   },
+  "iroha_data_model::ValueKind": {
+    "Enum": {
+      "variants": [
+        {
+          "name": "U32",
+          "discriminant": 0,
+          "ty": null
+        },
+        {
+          "name": "U128",
+          "discriminant": 1,
+          "ty": null
+        },
+        {
+          "name": "Bool",
+          "discriminant": 2,
+          "ty": null
+        },
+        {
+          "name": "String",
+          "discriminant": 3,
+          "ty": null
+        },
+        {
+          "name": "Name",
+          "discriminant": 4,
+          "ty": null
+        },
+        {
+          "name": "Fixed",
+          "discriminant": 5,
+          "ty": null
+        },
+        {
+          "name": "Vec",
+          "discriminant": 6,
+          "ty": null
+        },
+        {
+          "name": "LimitedMetadata",
+          "discriminant": 7,
+          "ty": null
+        },
+        {
+          "name": "Id",
+          "discriminant": 8,
+          "ty": null
+        },
+        {
+          "name": "Identifiable",
+          "discriminant": 9,
+          "ty": null
+        },
+        {
+          "name": "PublicKey",
+          "discriminant": 10,
+          "ty": null
+        },
+        {
+          "name": "Parameter",
+          "discriminant": 11,
+          "ty": null
+        },
+        {
+          "name": "SignatureCheckCondition",
+          "discriminant": 12,
+          "ty": null
+        },
+        {
+          "name": "TransactionValue",
+          "discriminant": 13,
+          "ty": null
+        },
+        {
+          "name": "TransactionQueryResult",
+          "discriminant": 14,
+          "ty": null
+        },
+        {
+          "name": "PermissionToken",
+          "discriminant": 15,
+          "ty": null
+        },
+        {
+          "name": "Hash",
+          "discriminant": 16,
+          "ty": null
+        },
+        {
+          "name": "Block",
+          "discriminant": 17,
+          "ty": null
+        },
+        {
+          "name": "BlockHeader",
+          "discriminant": 18,
+          "ty": null
+        }
+      ]
+    }
+  },
```

Comment:
Should we have these added in a separate commit, @s8sato? Or should we require that the commit contains an annotation?

Reply:
I don't think we need either. Commits that affected schema can be found by searching the git history of `schema.json`, instead of searching commit messages